### DISCUSSION TITLE
Experience and question "Anybody interested"

### DISCUSSION THREAD

**Discussion by Micharius** [MiczFlor/RPi-Jukebox-RFID#1473] [Q&A]
Hi there!

It took me a couple of hours, but my Phoniebox is now working! I had almost no idea about the Rasperry Pi, Linux, SSH etc. but now it works fine.

As I will use it only with Spotify as source, and being a general lazy person, I was thinking about to speed up the process of making the cards. 

So I made an Excel file, which helps to create the folders and files necessary, and also provides the source for labels:

First, I can scan the cards one by one in a column with the card reader attached to the computer. Then I can drag the album from Spotify webplayer to the adjacent column, and it will automatically extract the informations needed for the string which is necessary inside each _spotify.txt_ file. On another column, I paste the URL of each album or playlist cover picture.
I wrote some VBA procedure (I'm afraid, this is the only 'programming' I know 🤔) that creates the folders with the card names and also the _spotify.txt_ files inside. To make this working, the Phoniebox SMB Share must be assigned to a drive letter.

As an additional feature, I can use this Excel list to print labels: I have a word file that creates a merge document with the Excel file as source. I can now print labels with all the informations from the Excel list including the album picture for each card. This helps my daughter a lot to find the right cards to play.

I'm writing this because I thought somebody might be interested in the files I made. Please leave a comment, if you are! I could make the files a little bit more userfriendly and post them wherever you want. 

![screenshot_list](https://user-images.githubusercontent.com/88677184/128744301-5b5db5e1-fb34-4c45-87de-b71dc38093ec.png)

![screenshot_labels](https://user-images.githubusercontent.com/88677184/128744367-8c8f754e-d45e-46fe-8fad-c1a88d1a44e4.png)

And also, I would like to thank to the people that make this fascinating project possible!

Kind regards,

Micharius

**Comment by lenlennart** [comment]
Nice work, I am particularly interested in the second file (the one with the album covers). Are these automatically retrieved via Spotify API? Or are they gathered manually?

**Reply by Micharius** [reply]
Hi! 
No API, that would be to complicated for me 😒. 
You need to have the image's URLs for the album picture in a list such as excel. In my case, when I drag the albums from the Spotify webplayer to my list, I also copy the URL from the picture in a additional column. In Word, I have a custom field, which includes the image URL. When I create the labels in a new document, Word grabs the picture for every label based on the URL. If interested I can share you a sample file.
